### DISCUSSION TITLE
Setting up LDAPS authentication for airflow

### DISCUSSION THREAD

**Discussion by flyerhawk** [airflow-helm/charts#574] [Questions & Answers]
Hi,

I'm trying to get LDAPS auth set up for my airflow instance.  We do not allow ldap cleartext.     My challenge is figuring out how to get the flask app to use the proper cert.    The docs reference this value "AUTH_LDAP_TLS_CACERTFILE " but it is expecting a specific file location in the container.   Which means I would have to create a volume mount which is not ideal.   Is there a way to get that value to reference a K8s secret?

**Comment by zeddit** [comment]
hi, @flyerhawk have you configure the ldap correctly. I come into the trouble that airflow webserver cannot connect to ldap server. 
Our ldap server also not accepts the connection from insecure channel. 
could you share me with some ideas. thanks.
